packages/liquidsoap/liquidsoap.2.0.1/opam

```diff
@@ -89,13 +83,9 @@ conflicts: [
   "faad" {< "0.5.0"}
   "fdkaac" {< "0.3.1"}
   "ffmpeg" {< "1.1.0"}
+  "ffmpeg" {> "1.1.0"}
```

Comment:
```suggestion
  "ffmpeg" {>= "1.1.1"}
```
Using `>=` is usally always better than its stricter version as it would not allow for an eventual `1.1.0-1`